### PR TITLE
fix(types): Add index.d.ts to packaged files

### PR DESCRIPTION
**Pull request by forty**
Sorry, I did not notice earlier. Looks like we will need a 0.7.2 :grimacing:

### CODE REVIEW

**Comment by kachkaev** [comment]
Sorry for not spotting this when working on #283 😔

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "CHANGELOG.md",
     "LICENSE",
     "readme.md",
-    "lib"
+    "lib",
+    "index.d.ts"
```
Comment (brodybits) [inline package.json:27]: minor, optional: I would probably have reversed these:

```suggestion
    "index.d.ts",
    "lib"
```

- keep the non-documentation files alphabetical
- keep the interface file between the documentation and executable files

but this is just personal preference

**Review by brodybits — APPROVED** [review]
Approved on my part, with quick update to the title according to conventional commits

(please feel free to change it again if you like)

left a suggestion just from my personal preference

leaving up to @karfau for the next steps

**Comment by karfau** [comment]
Damn, I planned to double check exactly that, but forgot about it.
Thx for providing the fix.

Will proceed to release it

**Comment by kachkaev** [comment]
0.7.2 works perfectly 🔥

**Pull request merged**